cron/hourly/reconcile-transactions.js

```diff
@@ -15,9 +15,8 @@ import { processTransaction } from '../../server/paymentProviders/stripe/virtual
 const DRY = process.env.DRY;
 
 async function reconcileConnectedAccount(connectedAccount) {
-  const host = await models.Collective.findByPk(connectedAccount.CollectiveId);
-
-  const cards = await models.VirtualCard.findAll({ where: { HostCollectiveId: host.id } });
+  const host = connectedAccount.collective;
+  const cards = host.virtualCards;
```

Comment:
Doesn't solve the issue where we iterate over cards not related to the given connected account.